inspire/modules/workflows/tasks/submission.py

```diff
@@ -86,11 +87,17 @@ def _create_ticket(obj, eng):
         rt_queue = cfg.get("CFG_BIBCATALOG_QUEUES") or queue
         sip = d.get_latest_sip(sealed=False)
         subject = u"Your suggestion to INSPIRE: {0}".format(d.title)
+        try:
```

Comment:
I think this is fine as it is now. It should not be the case to have more than one ``hidden_note`` with ``source:submitter``